.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
```

Comment:
Every commit, travis will automatically try to tests. and report back on github what is the status
If there is a new tag, it will deploy it to a github release, and to an NPM release, hopefully

Reply:
See below, it shows a PR status